src/routes/team/_constants.js

```diff
@@ -0,0 +1,3 @@
+/* This file contains constants value specific to the team pages */
+export const current_year = 2020;
```

Comment:
Pls add a note in our README.md to change this every semester.

Reply:
Sure